File: hw/ramos/u1/hw07ChickenFarmSimulator/JavaScript/Egg.js
export default class Egg {
    #size;

    constructor(size) {
        this.#size = size;
    }

    getSize() { return this.#size; }
    setSize(size) { this.#size = size; }

    toString() {
        return `Egg{size=${this.#size}}`;
    }
}

File: hw/ramos/u1/hw14SafeInJsonChickenFarmSimulator/java/ChickenFarmSimulator/src/ec/edu/espe/ChickenFarmSimulator/model/Egg.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.ChickenFarmSimulator.model;

/**
 *
 * @author Paulo Ramos
 */
public class Egg {
    // Atributo privado
    private String size;

    // Constructor
    public Egg(String size) {
        this.size = size;
    }

    // Getter
    public String getSize() {
        return size;
    }

    // Setter
    public void setSize(String size) {
        this.size = size;
    }

    // Método toString sobrescrito
    @Override
    public String toString() {
        return "Egg{size=" + size + "}";
    }
}


